projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/unit-level-order/b2b-unit-level-order-history.e2e.cy.ts

```diff
@@ -137,7 +134,7 @@ describe('B2B - Unit-Level Orders History', { testIsolation: false }, () => {
   });
 
   describe('User who wants to sort order history', () => {
-    before(() => {
+    beforeEach(() => {
```

Comment:
needed as testisolation reset the navigation, test are now repeatable, important when a retry is needed.